src/framework/components/light/component.js

```diff
@@ -89,9 +89,9 @@ const _lightPropsDefault = [];
  * angle is specified in degrees. Affects spot lights only. Defaults to 40.
  * @property {number} outerConeAngle The angle at which the spotlight cone has faded to nothing.
  * The angle is specified in degrees. Affects spot lights only. Defaults to 45.
- * @property {number} lightSize Size of the light used for contact hardening shadows. For area lights
- * acts as a size multiplier with the area light dimensions. For punctual and directional lights
- * acts as the actual size of the light. Defaults to 1.0.
+ * @property {number} lightSize Size of penumbra for contact hardening shadows. For area lights
```

Comment:
I wonder if this should be called penumbraSize instead. @willeastcott - thoughts on this? As all this parameter affects is the penumbra size, but it the name suggests it could change the area light size too.

Reply:
If the description is `Size of penumbra`, then yes, I'd says `penumbraSize` is better. 😆